classes/Game.js

```diff
@@ -0,0 +1,92 @@
+import { Deck, Player, Grid } from './index'
+import _ from 'lodash'
+
+export default class Game {
+  constructor() {
+    this.setNewGame()
+    this.deal = this.deal.bind( this )
+  }
+
+  setNewGame() {
+    this.player = new Player()
+    this.deck = new Deck()
+    this.deck.cards = _.shuffle( this.deck.cards )
+    this.grid = new Grid()
+  }
+
+  deal() {
+  this.grid.fillNulls( this.deck )
+  }
+
+  handleSet() {
+    let cardsInPlay = this.grid.cardsInPlay
+    let selectedCards = this.player.selectedCards
+
+    if ( Game.isAValidSet( selectedCards ) === true ){
+      this.player.addScorePoint()
+
+      for (let i = 0; i < Game.CARDS_IN_PLAY_LENGTH; i++) {
+        for (let j = 0; j < Game.MAX_SET_LENGTH; j++) {
+          if ( _.isEqual(cardsInPlay[i], selectedCards[j]) ) {
+            cardsInPlay[i] = null
+          }
+        }
+      }
+      this.grid.cardsInPlay = cardsInPlay
+    }
+  }
+
+  reDeal() {
+    let deck = this.deck
+    let grid = this.grid
+
+    grid.cardsInPlay.forEach( cardInPlay => {
+      if ( cardInPlay.number === null ) {
+        return
+      }
+      deck.cards.push( cardInPlay )
+    })
+    this.grid.clear()
+    deck.cards = _.shuffle( deck.cards )
+    this.deck = deck
+    this.grid = grid
+    this.deal()
+  }
+
+
+
```

Comment:
I'm seeing extra whitespace used to break up different parts of your files - but it doesn't really increase the readability for me or indicate why they're separated. I'd suggest either splitting things you think need to be split into different modules or using comments as an alternative to whitespace to indicate why things are being separated.

In this instance, a comment to indicate the separation between prototype methods and instance methods would read better for me.